drive_cli/dcli.py

```diff
@@ -53,40 +54,41 @@ def go_back(picker):
 def drive_data(*argv):
     try:
         dclipath = os.path.join(dirpath, '.drivecli')
```

Comment:
Same comment as [this](https://github.com/nurdtechie98/drive-cli/pull/29/commits/fbc8b35c40ab6c5c43405e6336594f42405a9644#diff-ec2368969f827eb98adb4726413df627L29)